client/src/components/v3/status/WMStatus.tsx

```diff
@@ -22,6 +22,91 @@ export default function WMStatus(props: WMStatusProps) {
     </span>
   );
 
+  function convertToSentenceCase(str: String) {
+    /**
+     * convert Camel Case words to sentence case
+     *
+     * @param str - camel case string
+     * @returns string in sentence format
+     */
+    const result = str.replace(/([A-Z])/g, ' $1');
+    const final = result.charAt(0).toUpperCase() + result.slice(1);
+    return final;
+  }
+
+  function extractData(type: string, data: any) {
+    /**
+     * extract data from JSON objects and present them on client side so its user friendly
+     *
+     * @param type -  type of the data
+     * @param data - data JSON received from the mqtt broker
+     * @returns JSON-string
+     */
+
+    function formatValue(value: any, unit: any) {
+      /**
+       * receives a value and it unit and format them appropriately
+       *
+       * @param value - any value
+       * @param unit
+       * @returns string containing hte value and its unit
+       */
+      let displayValue;
+      if (value !== null && value !== undefined) {
+        displayValue = Math.floor(value * 100) / 100;
+      } else {
+        displayValue = '-';
+      }
+      const displayUnit = unit ? ` ${unit}` : '';
+
+      return `${displayValue}${displayUnit}`;
+    }
+
+    if (type === 'gps') {
+      return (
+        <>
```

Comment:
We only need these "fragments" if we're returning multiple top-level JSX objects (e.g. `<table>`). In this case, they're not required.